### PR TITLE
RAC-5128 - modify workflows test to use new AMQP

### PR DESCRIPTION
**Pull request by larry-dean**

### CODE REVIEW

**Comment by larry-dean** [comment]
@hohene @johren @stuart-stanley

File: test/tests/api-cit/v2_0/workflows_tests.py [hunk 1]
```diff
@@ -192,90 +211,55 @@ def post_workflows(self, graph_name, timeout_sec=10, nodes=[], data=None, tasks=
...
+                # make sure to cannel any possible active workflow
+                self.__cancel_active_workflow(node_id)
+
+                # post new workflow
+                Api().nodes_post_workflow_by_id(node_id, name=self.__class__.__graph_name, body=data)
+                logs.info("Posted workflow %s on node %s", self.__class__.__graph_name, node_id)
+                data = self.__get_data()
+                workflow_id = data['instanceId']
+
+                logs.info('******************** wf_id: %s', workflow_id)
```
Comment (stuart-stanley) [inline test/tests/api-cit/v2_0/workflows_tests.py]: nit: I recall throwing these in with all the '*'s in order to find them in the logs easier. I don't think they should stay way? (and might as well put them both in one line!)

File: test/tests/api-cit/v2_0/workflows_tests.py [hunk 2]
```diff
@@ -154,35 +166,42 @@ def test_workflows_graphs_put(self):
...
         Api().workflows_delete_graphs_by_name(self.workflowDict.get('injectableName'))
-        self.assertEqual(204, self.__client.last_response.status)
+        self.assertEqual(204, self.__get_result().status)
         Api().workflows_get_graphs_by_name(self.workflowDict.get('injectableName'))
-        self.assertEqual(0, len(loads(self.__client.last_response.data)))
+        self.assertEqual(0, len(self.__get_data()))
+
+    @depends(after=['test_workflows_graphs_name_put', 'test_delete_all_active_workflows'])
+    def test_node_workflows_post(self):
+        logs.info('************************* test_node_workflows_post ')
```
Comment (stuart-stanley) [inline test/tests/api-cit/v2_0/workflows_tests.py]: nit: I recall throwing these in with all the '*'s in order to find them in the logs easier. I don't think they should stay way?

**Comment by stuart-stanley** [comment]
test this please

**Comment by larry-dean** [comment]
test this please

**Comment by larry-dean** [comment]
Test this please

**Pull request merged**